<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DailySummaryDialog</class>
 <widget class="QDialog" name="DailySummaryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Günlük Özet Raporu Aracı</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_Settings">
     <property name="title">
      <string>Ayarlar</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_DateRange">
        <property name="text">
         <string>Tarih Aralığı:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_Dates">
        <item>
         <widget class="QDateEdit" name="date_Start">
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_to">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="date_End">
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_DateColumn">
        <property name="text">
         <string>Tarih Sütunu:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="combo_DateColumn">
        <property name="toolTip">
         <string>Verilerin gün bazında gruplanacağı tarih sütunu (örn: TARIH)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_DataColumn">
        <property name="text">
         <string>Veri Sütunu:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="combo_DataColumn">
        <property name="toolTip">
         <string>İşlem yapılacak sayısal veri sütunu (örn: H_DEBI1)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_AggType">
        <property name="text">
         <string>İşlem Türü:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="combo_AggType">
        <item>
         <property name="text">
          <string>Günlük Toplam (Sum)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Günlük Ortalama (Average)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Günlük Fark (Maksimum - Minimum)</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_RunSummary">
     <property name="text">
      <string>Özeti Oluştur</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_Results">
     <property name="title">
      <string>Sonuçlar</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableWidget" name="table_SummaryResults">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_ExportExcel">
        <property name="text">
         <string>Sonucu Excel'e Aktar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>